pkg/cmd/pulumi/state/state.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016-2024, Pulumi Corporation.
+// Copyright 2025, Pulumi Corporation.
```

Comment:
(Nit)
```suggestion
// Copyright 2016-2025, Pulumi Corporation.
```
In general -- for new files, our copyright over them starts and ends today (this year). For files that have existed longer, they have been under our copyright as long as we have been maintaining them.